packages/react/src/pill/Pill.tsx

```diff
@@ -0,0 +1,70 @@
+import { type KeyboardEvent, type ReactNode, forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+import { Button } from "../button";
+import { IconX } from "../icons/IconX";
+import { extractSprinkles } from "../sprinkles";
+import { Text } from "../text";
+// import { Text } from "../text";
+import * as styles from "./Pill.css";
+
+type PillProps = BoxProps<
+  "button",
+  {
+    endDecorator?: ReactNode;
+    onRemove?: () => void;
+    startDecorator?: ReactNode;
+  } & styles.PillVariants
+>;
+
+export const Pill = forwardRef<HTMLButtonElement, PillProps>(
+  (
+    {
+      children,
+      className,
+      endDecorator,
+      onRemove,
+      size = "lg",
```

Comment:
Let's keep `md` as default, as we are following this size on others component also as default.